src/fides/api/alembic/migrations/helpers/database_functions.py

```diff
@@ -0,0 +1,6 @@
+import uuid
+
+
+def generate_record_id(prefix):
+    """Generates an ID that can be used for a database table row ID."""
+    return prefix + "_" + str(uuid.uuid4())
```

Comment:
I pulled this out into a helpers module but I'm not sure if this would be considered "bad practice" since changes here could change the original intention of the migration scripts that use this. This is a general concern also for the functions in `fideslang_migration_functions.py`. Open to suggestions here!